### PR TITLE
US152268 - Migrate rest of filter vdiff tests

### PR DESCRIPTION
**Pull request by svanherk**
This PR does three things:

- Removes the forced "opening-up" functionality of the `d2l-filter` vdiff tests. I expect diffs for each that were opening up and are now opening down, and their mobile counterparts because of opener location change.
- Migrates `filter-overflow-group` vdiff tests. Not expecting big changes, will comment on the goldens PR.
- Migrates `filter-tags` vdiff tests. Not expecting big changes other than some browser integration differences, will comment on the goldens PR.

Final Report: https://vdiff.d2l.dev/BrightspaceUI/core/7671b327d8b3e61a50e842b343cac2588610aa8b/2023-09-15_13-40-01/report/

### CODE REVIEW

File: components/filter/test/filter-tags.vdiff.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+	describe('clearable behavior', () => {
+		it('deleting the last item', async() => {
+			const elem = await fixture(tagsSingleFilter, { viewport: { width: 1700 } });
+			const filterTags = elem.querySelector('d2l-filter-tags');
+			const items = filterTags.shadowRoot.querySelectorAll('d2l-tag-list-item');
+			const deleteButton = items[6].shadowRoot.querySelector('d2l-button-icon');
+
+			await clickElem(deleteButton);
+			await aTimeout('300'); // clearing has a 200ms delay
```
Comment (dlockhart) [inline components/filter/test/filter-tags.vdiff.js:114]: These delays are unfortunate but unavoidable I guess eh.

**Review by dlockhart — APPROVED** [review]
Looks great!

File: components/filter/test/filter-tags.vdiff.js (hunk 1 above)
Comment (svanherk) [reply components/filter/test/filter-tags.vdiff.js:114]: Hmm, I can try playing around with sinon timer stuff

Comment (svanherk) [reply components/filter/test/filter-tags.vdiff.js:114]: Actually, I'm gonna merge this as is and come back to this if I have time. I'm running into issues with mocking the timers affecting the other stuff that needs to run in the test. We could also consider allowing this timeout to be configurable for tests

**Pull request merged**